bytecode/tests/__init__.py

```diff
@@ -113,7 +113,7 @@ def dump_bytecode(code, lineno=False):
             print()
 
 
-def get_code(source, *, filename="<string>", function=False):
```

Comment:
This is one good reason to not support Python 2: it forces to write crappy API :-)